maintainers/gpuci.md

```diff
@@ -269,3 +233,23 @@ The script should calculate the next version and update the source code, build s
 **Note:** The script must export `NEXT_FULL_TAG` as the full `x.y.z` version.
 
 After this, gpuCI will commit the changes and tag the commit with `v${NEXT_FULL_TAG}` before pushing to the repository.
+
+## Environment variables
+
+gpuCI exposes various environment variables that the CI scripts can utilize.
+
+`BUILD_MODE` is either `pull-request` or `branch` for PR builds and branch builds respectively.
```

Comment:
For information like 'nightly build', which of these env vars should be used?

Reply:
@tijyojwad branch builds are the builds that are triggered upon a merge commit. These are the same builds that upload our conda packages and do full gpu matrix testing. "Nightly" is a misnomer, but they refer to our branch builds.